src/tilemap_manager/errors.rs

```diff
@@ -19,3 +19,9 @@ pub enum TilemapManagerError {
     #[error("TileData does not exist for the given ChunkCell")]
     TileDataDoesNotExist,
 }
+
+impl<'w> From<QueryEntityError<'w>> for TilemapManagerError {
```

Comment:
Can you explain what this change does? Is there a reason to not use the #[from] generated impl?

Reply:
Sorry, I missed this comment! It was because we can't use the generated version, due to the change in lifetimes in Bevy 0.15 (a bunch of things went to elided lifetimes which subtly complicated matters in ways I don't fully understand). There might be a better solution, I just couldn't find it!